app/src/components/Tabs/my-inventories-tab.tsx

```diff
@@ -178,10 +180,12 @@ const MyInventoriesTab: FC<MyInventoriesTabProps> = ({
                   >
                     <Box>
                       <Box display="flex" gap="8px" alignItems="center">
-                        <Avatar
-                          className="h-[32px] w-[32px]"
-                          name="Argentina"
-                          src="https://upload.wikimedia.org/wikipedia/commons/1/1a/Flag_of_Argentina.svg"
+                        <CircleFlag
+                          countryCode={extractCountryLocode(
+                            city.countryLocode!,
```

Comment:
You are passing the country locode, not the city locode to this function. So I think the call to extractCountryLocode is not required or you need to pass the city locode into it...